aws-android-sdk-iot-test/src/androidTest/java/com/amazonaws/mobileconnectors/iot/IotClient.java

```diff
@@ -254,6 +234,48 @@ boolean deletePolicy(String policyName) {
         }
     }
 
+    void createAuthorizer(
+            @NonNull String authorizerName,
+            @NonNull String authorizerFunctionArn,
+            @NonNull String tokenKeyName,
+            @NonNull String publicKey) {
+        Objects.requireNonNull(authorizerName);
+        Objects.requireNonNull(authorizerFunctionArn);
+        Objects.requireNonNull(tokenKeyName);
+        Objects.requireNonNull(publicKey);
+
+        CreateAuthorizerRequest request = new CreateAuthorizerRequest();
+        request.setSigningDisabled(false);
+        request.setAuthorizerName(authorizerName);
+        request.setAuthorizerFunctionArn(authorizerFunctionArn);
+        request.setTokenKeyName(tokenKeyName);
+        request.setStatus(AuthorizerStatus.ACTIVE);
+        request.setTokenSigningPublicKeys(Collections.singletonMap("FIRST_KEY", publicKey));
+        iot.createAuthorizer(request);
+    }
+
+    void cleanupAuthorizer(String authorizerName) {
+        try {
+            updateAuthorizer(authorizerName, AuthorizerStatus.INACTIVE);
+            deleteAuthorizer(authorizerName);
+        } catch (ResourceNotFoundException noSuchAuthorizer) {
+            // Oh well.
```

Comment:
Win some, lose some.

Reply:
Ill do the `true`/`false` boolean return thing here, so the info is not lost.